__init__.py

```diff
@@ -242,8 +242,10 @@ def __add_key_and_cookie_file(self,
             my_pass = get_linux_pass(os_crypt_name)
 
             iterations = 1
-            self.key = PBKDF2(my_pass, self.salt,
-                              iterations=iterations).read(self.length)
+            self.v10_key = PBKDF2(b'peanuts', self.salt,
```

Comment:
Adding `v10_key` on the Linux case only will raise an error later in ` _decrypt()`. 
Please add `v10_key` at the at the top of `__add_key_and_cookie_file()` so that it covers all the OSes. 
Then add `v11_key` per a OS basis.